client/src/components/dollarsignsandCategories.jsx

```diff
@@ -2,25 +2,90 @@ import styled, { css } from 'styled-components';
 import React from 'react'
 
 const Dollars = styled.div`
-  line-height: 21px;
   color: #666;
 `
 
 const DollarsDot = styled.i`
-  font-size: 0.1rem;
+  font-size: 0.05rem;
   display: inline;
   color: #666666;
   top: -10px;
-  margin: 1px;
-  padding-left: 3px;
+  padding: 0px 8px;
   vertical-align: middle;
 `
 
 const CategoryLink = styled.a`
   color: #0073BB;
   text-decoration: none;
+  &:hover{
+    text-decoration: underline;
+  }
+`
+CategoryLink.displayName = "CateogryLink"
+
+const EditButton = styled.button`
+  border-radius: 3px;
+  border-width: 1px;
+  padding: 3px 5px;
+  margin-left: 10px;
+  border-style: solid;
+  border-color: #ccc;
+  color: #A3A3A3;
+  background-color: #F5F5F5;
+  font-size: .7rem;
+`
+const EditLink = styled.a`
+  text-decoration: none;
+  color: #A3A3A3;
+  position: relative; 
+  &:hover #ToolTip{
+    visibility: visible;
+  }
+  &:hover{
+    color: black;
+  }
+`
+const EditIcon = styled.i`
+  padding-right: 2px;
+`
+
+const ToolTip = styled.span`
+  visibility: hidden;
+  width: 90px;
+  background-color: black;
+  color: #fff;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  opacity: 0.9;
+
+  position: absolute;
+  z-index: 1;
+  top: -35px;
+  left: -25px;
+  &:after {
+    content: "";
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    margin-left: -5px;
+    border-width: 5px;
+    border-style: solid;
+    border-color: black transparent transparent transparent;
```

Comment:
any reason for not just specifying a border-top?

Reply:
Yes - I need to specify that the other borders are transparent or else they would be white, which is not what i need to make this tool tip work. It seemed easier this way to write them.

Reply:
cool. figured there was a reason, just wasn't sure of it